projects/libhtp/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER victor@inliniac.net
+RUN apt-get update && apt-get install -y make autoconf automake libtool zlib1g-dev
+RUN git clone --depth 1 https://github.com/OISF/libhtp.git libhtp
+WORKDIR $SRC
+COPY build.sh $SRC/
+#TODO remove
+COPY fuzz.diff $SRC/fuzz.diff
```

Comment:
This won't work right? Same for build.sh:21.
Please do a build locally before you want this merged.

Reply:
Thanks @jonathanmetzman I removed this dead code as I had commented
Sorry for having hurried too much, I will take good resolutions next year

Reply:
And I just tested it and it works locally (finds one integer overflow with undefined sanitizer)

Reply:
Thanks! Happy Holidays!